packages/contracts-sdk/src/IncentivesFundManager.types.ts

```
@@ -0,0 +1,38 @@
+  native_token: {
+    denom: string;
+  };
+};
+export interface Asset {
+  amount: Uint128;
+  info: AssetInfo;
+}
+export type QueryMsg = {
+  config: {};
+};
+export interface MigrateMsg {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)